packages/app-project/stores/Collections.js

```diff
@@ -0,0 +1,108 @@
+import { autorun } from 'mobx'
+import { addDisposer, flow, getRoot, types } from 'mobx-state-tree'
+import auth from 'panoptes-client/lib/auth'
+import asyncStates from '@zooniverse/async-states'
+
+export const Collection = types
+  .model('Collection', {
+    display_name: types.maybeNull(types.string),
+    favorite: false,
+    id: types.identifier,
+    links: types.maybeNull(types.frozen({})),
+    private: true
+  })
+
+const Collections = types
+  .model('Collections', {
+    error: types.maybeNull(types.frozen({})),
+    favourites: types.maybeNull(Collection),
```

Comment:
Should this be `collections` or maybe `resources` to be consistent with how the stores are setup in the classifier? Favorites are just a type of collection and not what the model is called on Panoptes. I'd like to have the models in our stores match as closely to the Panoptes names.

Reply:
I’d like to store favourites separately from other collections, since that collection gets read from/written to by the UI. I’m reserving collections/resources for a list of your other collections on the project, if we need to store that.

Reply:
I’d also add that the models here should match the business logic of the JS app. I can see the advantage in keeping names consistent but it might not be a good idea to tightly couple the model here to the database model.